source/_docs/automation/editor.markdown

```diff
@@ -31,3 +31,7 @@ message: Sensor value greater than 10
 
 Automation created or edited via the user interface, are activated immediately
 after save the automation.
+
+## Troubleshooting missing automations
+
+Some users may find that when creating automations using the GUI, they appear to be created but never show in the automations page. When creating automations using the GUI, they are written to `automations.yaml` even if you have a [split configuration](/docs/configuration/splitting_configuration/#example-combine-include_dir_merge_list-with-automationsyaml). Make sure you are correctly including the base `automations.yaml` file to ensure they are loaded.
```

Comment:
> Some users

We write documentation directly to the reader, thus "If you" is more fitting.

Reply:
Fair point.